### ISSUE TITLE
OpenGL Window Creation

### ISSUE THREAD

**Issue by AnorexicAtticusFinch** [WebClub-NITK/Hacktoberfest-2k19#153]
### Description
Create an OpenGL context and an application window to render in using SDL, GLUT or GLFW.

### Details
- Technical Specifications: C/C++
- Type of issue: Single
- Time Limit: 2 days

### Issue requirements / progress

- [ ] - Open a window
- [ ] - Allow the window to be resizable
- [ ] - Make the window a solid color of your choice (Not Black)

### Resources
https://learnopengl.com/Getting-started/OpenGL

### Directory Structure
Make sure a directory is created under the corresponding domain folder properly following the collaborator guidelines and mention it here.

#### Note
Please claim the issue first by commenting here before starting to work on it.
While submitting a PR, please request a review from @AnorexicAtticusFinch

**Comment by IvanFLeon** [comment]
I would like to give this a try!

**Comment by AnorexicAtticusFinch** [comment]
@IvanFleon you have a day left, have you made any progress?

**Comment by IvanFLeon** [comment]
Yes! I am already done. Under what directory and name should I add the file?

**Comment by AnorexicAtticusFinch** [comment]
Save it using the issue number under the 'systems' folder. Follow the instructions mentioned in the guidelines.

**Comment by AnorexicAtticusFinch** [comment]
You haven't opened an OpenGL context, you have only opened a window using GLFW. You need to use the gladLoadGLLoader function (in the glad.c file, which you also need to include) to load the OpenGL processes during runtime. You also need to use the glViewport function for properly resizing the window, without it the window would be resized but the OpenGL render target would not be made aware of this and would continue to render at the old size. Refer to the link under the resources section of the issue, it covers all of these things.

**Comment by IvanFLeon** [comment]
I've added the missing features. Let me know if there is anything that needs to be changed or added.

**Comment by AnorexicAtticusFinch** [comment]
It's right, nothing else to be done!
@SaurabhAgarwala this can be merged

**Comment by SaurabhAgarwala** [comment]
#256 closes this
